backend/src/database/repositories/memberRepository.ts

```diff
@@ -374,15 +372,24 @@ class MemberRepository {
               inner join identities i on i."memberId" = m.id
       where mi."tenantId" = :tenantId
         and mi.platform = :platform
-        and mi.username = :username
+        and mi.username in (:usernames)
     `
 
+    const usernames: string[] = []
+    if (typeof username === 'string') {
+      usernames.push(username)
+    } else if (Array.isArray(username)) {
+      usernames.push(...username)
+    } else {
+      throw new Error('Unknown username format!')
```

Comment:
Can we return a list of the formats we allow?

Reply:
Yes - will add in the error message.